Fluid.MvcViewEngine/FluidRendering.cs

```diff
@@ -62,6 +62,9 @@ static FluidRendering()
             context.AmbientValues[ViewPath] = path;
             context.AmbientValues.Add("Sections", new Dictionary<string, IList<Statement>>());
 
+            var viewsPath = Path.Combine(_hostingEnvironment.ContentRootPath, "Views");
```

Comment:
At first glance the name is confusing little bit, but it make sense because it map from filepath to `{Partial Folder}/filepath` 😄